src/components/snapshots/routes/routeUploadSnapshot.ts

```diff
@@ -32,14 +32,20 @@ const routeUploadSnapshot = async (request: FastifyRequest, reply: FastifyReply)
     for await (const file of data) {
       await pump(file.file, fs.createWriteStream(snapshotZipLocation))
 
-      // Below syncrhonous opperations are probably ok for snapshot routes
+      // Below synchronous opperations are probably ok for snapshot routes
       const zip = new zipper(snapshotZipLocation)
       const snapshotMainFileName = `${SNAPSHOT_FILE_NAME}.json`
 
-      if (!zip.getEntries().find(({ entryName }) => entryName === snapshotMainFileName)) {
+      if (
+        !zip
+          .getEntries()
+          .find(
+            ({ entryName }) => entryName === snapshotMainFileName || entryName === 'database.dump'
+          )
+      ) {
```

Comment:
Just changing what gets recognised as a valid snapshot. Previously it looked for snapshot.json, now it'll accept either snapshot.json or database.dump (should have been part of last PR, really)